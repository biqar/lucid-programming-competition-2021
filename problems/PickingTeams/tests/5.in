20
+ Alice Bob
+ Charlie David
? Alice Bob
? Alice David
+ Bob David
? Alice David
+ Charlie David
? Alice David
? Charlie David
+ Charlie Emily
+ Emily Francis
+ Francis Ginny
+ Alice Emily
? Alice Emily
? Alice Francis
? Alice Ginny
? Charlie Ginny
+ Charlie Francis
? Charlie Francis
? Alice Francis
