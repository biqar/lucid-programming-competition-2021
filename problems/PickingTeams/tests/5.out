yes
no
yes
yes
no
yes
yes
yes
no
no
yes
2
